File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.py
class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        x= list(set(nums))
        x.sort()
        x_len=len(x)
        l=['_']*(len(nums)-x_len)
        nums[::]=x+l
        return x_len
            

File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.cpp
class Solution {
public:
    int removeDuplicates(vector<int>& nums) {
        int n=nums.size();
        int j=1;
        for(int i=1;i<n;i++){
            if(nums[i-1]==nums[i]) continue;
            nums[j]= nums[i];
            j++;
        }
            return j;

        }
    }; 